dissect/target/plugins/os/unix/_os.py

```diff
@@ -227,7 +234,20 @@ def _parse_hosts_string(self, paths: list[str] | None = None) -> dict[str, str]:
     def _add_mounts(self) -> None:
         fstab = self.target.fs.path("/etc/fstab")
 
-        for dev_id, volume_name, mount_point, _, options in parse_fstab(fstab, self.target.log):
+        for dev_id, volume_name, mount_point, fs_type, options in parse_fstab(fstab, self.target.log):
+            # Mount nfs, but only when target has been mapped by the `LocalLoader`
+            if fs_type == "nfs" and isinstance(self.target._loader, LocalLoader):
+                if "enable-nfs" in self.target.path_query:
+                    self._add_nfs(dev_id, volume_name, mount_point)
+                else:
+                    log.warning(
+                        "NFS mount %s:%s at %s is disabled. To enable, pass --enable-nfs to the local loader. Alternatively, add a query parameter to the target query string: local?force-directory-fs&enable-nfs",  # noqa: E501
```

Comment:
Don't think you need the `force-directory-fs`?

Reply:
It's an example, I can remove it.

Reply:
I think an example like that should be in the docs or `-h`, but in the log message it should be clear what exactly is necessary for the logged-about feature to function.

Reply:
I will create a ticket / check if there is already one to streamline the passing of arguments to loaders.

Or perhaps @EinatFox can do that,

Reply:
Can tie into https://github.com/fox-it/dissect.target/issues/920 as well.